packages/normalizr/src/schemas/Object.js

```diff
@@ -52,6 +52,14 @@ export const denormalize = (schema, input, unvisit) => {
   return [object, found, deleted];
 };
 
+export function infer(schema, args, indexes, recurse) {
+  let resultObject = {};
+  for (const k in schema) {
```

Comment:
presumably you want to iterate Object.keys(schema) here, rather that using for-in?